src/script/main/app.js

```diff
@@ -114,7 +115,7 @@ class App {
     repositories.auth = authComponent.repository;
     repositories.giphy = resolve(graph.GiphyRepository);
     repositories.properties = new PropertiesRepository(this.service.properties, resolve(graph.SelfService));
-    repositories.serverTime = new z.time.ServerTimeRepository();
+    repositories.serverTime = new ServerTimeRepository();
```

Comment:
```suggestion
    repositories.serverTime = resolve(graph.ServerTimeRepository);
```

And you can remove the `import` at the top of the file